package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@cparra/apexdocs",
-  "version": "3.4.2",
+  "version": "3.4.3",
```

Comment:
I'll take care of the release so let's revert this. Since this Apexdocs is following semver the next one will be `3.5.0` as I consider this change to be an additional feature and not a bug fix.